plugins/reporters/aosd/src/funTest/assets/aosd21/aosd.NPM-%40ort-project-with-findings-1.0.json

```diff
@@ -11,8 +11,8 @@
   ],
   "components": [
     {
-      "id": 0,
-      "componentName": "no-license-file",
+      "id": 3,
```

Comment:
commit message: Why is it favorable to align with [1] ?

Reply:
As said in the title: To more easily compare both JSON output files with a diff tool, while focusing on real differences.

Reply:
I had read that title of course. To me it wasn't clear the motivation behind being able to compare to that tool?

Reply:
I've also added to the commit message that `spdx2aosd` actually is a tool that's officially used by Audi.